odk1-src/collect-form-map.rst

```diff
@@ -0,0 +1,36 @@
+.. spelling::
+
+  unfinalized
+  basemap
+
+Viewing filled forms on a map
+================================
+
+.. versionadded:: 1.25
```

Comment:
Ah this is a really nice way of showing where a feature is added. Totally stealing this.